### PR TITLE
Different VSIX for VS 15.0.

### PR DESCRIPTION
**Pull request by deepakaravindr**
1. Changed the VSIX ID.
2. Changed the display name
3. Changed the supported VS Version to be 15.0.

@yishaigalatzer @rrelyea

### CODE REVIEW

**Comment by AlexEyler** [comment]
Looks good here as well (Extensibility team member)

**Comment by rrelyea** [comment]
:shipit:

File: src/NuGet.Clients/VsExtension/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
     <PreviewImage>Resources/nuget_256.png</PreviewImage>
   </Metadata>
   <Installation InstalledByMsi="false" AllUsers="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VWDExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VPDExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VSWinExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VSWinDesktopExpress" Version="[14.0,]" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="15.0" />
```
Comment (AArnott) [inline src/NuGet.Clients/VsExtension/source.extension.vsixmanifest]: Should you use `[15.0,16.0)` to start in a more defensive posture? You can always release an updated VSIX that allows 16.x use after you've tested it.
Or perhaps simply specifying 15.0 has the same effect, I don't know.

Comment (deepakaravindr) [inline src/NuGet.Clients/VsExtension/source.extension.vsixmanifest]: I don't know either. Following up on it...

**Comment by deepakaravindr** [comment]
I have updated the PR with the comment on display name.
Will not merge yet. There is an open comment on the version range.

**Comment by AArnott** [comment]
Looks good to me, provided you've re-tested the VSIX's willingness to install to Dev14 and Dev15 after the latest change.

**Comment by deepakaravindr** [comment]
Merged with https://github.com/NuGet/NuGet.Client/commit/b56187b7d25263a061ffa916d17491ef8cda171e

**Pull request closed without merging**